charter.md

```diff
@@ -0,0 +1,22 @@
+# Open Web Docs Charter
+
+## Mission
+
+Open Web Docs supports web platform documentation for the benefit of web developers & designers worldwide.
+
+
+## Core Values
+
+Open Web Docs believes open web platform documentation is critical digital infrastructure that benefits everyone. To demonstrate that belief, we: 
+
+- _focus_ on documentation that has a positive impact on end-users and web developers.
+- _prioritize_ based on needs of the global community of web developers and designers. 
+- _maintain_ an independent editorial voice, with a “firewall” between fiscal and editorial matters. 
+- _adhere_ to open source principles of transparency in project governance, investment, and feature development in order to maintain trustworthiness and credibility with the web community. 
+- _work_ to make the platform more equitable, investing in documenting internationalization, accessibility, privacy and diversity components of the web. 
+- _emphasize_ diversity and inclusion across all our efforts, including project contributions, community organization, and resource allocation.
+
+## What We Do
+To execute our mission and values, we contribute to MDN and other related efforts with an independent editorial voice, rooted in the web technology landscape reflecting the lived experience of web developers. We take a lead role on specific content projects within these platforms, in coordination with these platforms where appropriate.
```

Comment:
"MDN Web Docs and other related documentation platforms..."

Reply:
"...rooted in the web technology landscape reflecting..." -> "...rooted in the web technology landscape and reflecting..."

Reply:
"...within these platforms, in coordination with these platforms where appropriate." -> "...within these platforms, in coordination with their core teams where appropriate." ?